pycqed/tests/test_spectroscopy_analysis.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+import pycqed as pq
+import os
+from pycqed.analysis import measurement_analysis as ma
+
+
+class Test_Homodyne_analysis(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(self):
+        self.datadir = os.path.join(pq.__path__[0], 'tests', 'test_data')
+        ma.a_tools.datadir = self.datadir
+
+    def test_Homodyne_analysis_hanger(self):
+        #lorentzian model tested in test_heterodyne_analysis.py
+
+        a = ma.Homodyne_Analysis(timestamp='20170929_174145', label='resonator_spec')
+        try:
```

Comment:
Try except statements in a test don't sound like a very good idea. 
Please choose only one case that passes and do not use try/except. .